k8s/plugin/src/resources/mod.rs

```diff
@@ -20,6 +28,42 @@ pub enum Operations {
     /// 'Uncordon' resources.
     #[clap(subcommand)]
     Uncordon(CordonResources),
+    /// 'Upgrade' resources
+    #[clap(subcommand)]
+    Upgrade(GetUpgradeResources),
```

Comment:
hmm I don't this is quite up to the current implementation.
Although I prefer what you did, let's keep with the existing pattern until we change everything, so we'd have all the verbs (Install,Apply,Get,Uninstall) at the root.